api/auth/session.js

```diff
@@ -90,14 +74,7 @@ module.exports = ({ Cookies = defaultCookies } = {}) => {
         // an expiration date.
         cookies.set(
           COOKIE_NAME,
-          jwt.sign(
-            { iss: 'CMS eAPD API', payload: session.content },
-            process.env.SESSION_SECRET,
-            {
-              algorithm: 'HS256',
-              expiresIn: `${process.env.SESSION_LIFETIME_MINUTES}m`
-            }
-          ),
+          signWebToken({ payload: session.content }),
```

Comment:
Extracted common functionality in this file into functions (`signWebToken`, `verifyWebToken`)  within `api/auth/jwtUtils.js`